### DISCUSSION TITLE
Bombersnake

### DISCUSSION THREAD

**Discussion by nathanlambert** [BattlesnakeOfficial/feedback#21] [General]
Standard Battlesnake rules, but snakes can drop a bomb every X moves from its tail. 3 or so moves later the bombs blow up in a `+` similar to Bomberman and kill any other snakes if the bomb hits their head.

The bomb radius could lengthen over time if a snake doesn't drop any, or power-ups could randomly spawn to give the bomb more radius.

**Comment by andrewdmaclean** [comment]
Hey @nathanlambert I just wanted to let you know that we definitely have this on our list of feature requests to look into!
